src/presets/netlify.ts

```diff
@@ -89,12 +89,18 @@ async function writeRedirects(nitro: Nitro) {
   );
 
   // Rewrite static cached paths to builder functions
-  for (const [key] of rules.filter(
-    ([_, routeRules]) =>
-      routeRules.cache && (routeRules.cache?.static || routeRules.cache?.swr)
+  for (const [key, value] of rules.filter(
+    ([_, value]) =>
+      value.cache === false ||
+      (value.cache && value.cache.swr === false) ||
+      (value.cache && (value.cache?.static || value.cache?.swr))
```

Comment:
I think this can be simplified for swr check only.

---

Asking ChatGPT: 🤓 

<img width="920" alt="image" src="https://user-images.githubusercontent.com/5158436/211801354-e880c3b5-33a7-4e4e-8221-8a7cc7be932d.png">

<img width="777" alt="image" src="https://user-images.githubusercontent.com/5158436/211801382-1a4fca4d-6332-4c9e-a3d7-fad2b8d771b1.png">

Reply:
Yeah. The main problem is that we are not deeply normalizing the route rules at the moment and with _undifined_ conditions, such simplification is wrong. Cache should be either `false` or an object. And I'm thinking the static flag could be moved outside. Lets improve later.